test/runtime/jhh/colocales/colocales.py

```diff
@@ -264,7 +277,10 @@ def main(argv):
     compiler = argv[1]
     os.environ['CHPL_COMPILER'] = compiler
     del argv[1]
-    localDir = sub_test.get_local_dir(sub_test.get_chpl_base(compiler))
+    baseDir = sub_test.get_chpl_base(compiler)
+    homeDir = sub_test.get_chpl_home(baseDir)
+    testDir = sub_test.get_test_dir(homeDir)
+    localDir = sub_test.get_local_dir(testDir)
```

Comment:
What are these new unused variables for? Do the `sub_test.get_*` calls do printing? If so, maybe just have the calls and not assign to an unused variable?

Reply:
Each variable is used in the next line. I did this to make it clearer what is going on, vs. using the same variable for each call or nesting the calls.